corehq/apps/app_manager/models.py

```diff
@@ -2030,6 +2030,10 @@ def from_json(cls, data):
                     item.value[lang] = interpolate_media_path(path)
         return to_ret
 
+    @property
```

Comment:
it usually prefer having such checks as methods on objects instead of doing the string comparison at various places. So in case the value is to be changed someday, it needs to be done only at one place. For ex here if 'invisible' is changed to 'hidden'.
This made more sense in ruby where method name would actually be 'invisible?' but i think it still holds value here.